saos-webapp/src/main/resources/message/common.properties

```diff
@@ -114,20 +115,23 @@ judgmentSearch.formField.commonCourt = Sąd
 judgmentSearch.formField.commonDivision = Wydział
 judgmentSearch.formField.supremeChamber = Izba sądu najwyższego
 judgmentSearch.formField.supremeChamberDivision = Wydział izby sądu najwyższego
-judgmentSearch.formField.chooseCommonCourt = Wybierz Sąd
-judgmentSearch.formField.chooseCcDivision = Wybierz wydział
-judgmentSearch.formField.chooseSupremeChamber = Wybierz izbę
-judgmentSearch.formField.chooseSupremeChamberDivision = Wybierz wydział izby
-judgmentSearch.formField.chooseScJudgmentForm = Wybierz formę orzeczenia
-judgmentSearch.formField.chooseScPersonnelType = Wybierz typ składu sędziowskiego
+judgmentSearch.formField.chooseCommonCourt = Dowolny sąd
+judgmentSearch.formField.chooseCcDivision = Dowolny wydział
+judgmentSearch.formField.chooseSupremeChamber = Dowolna izba
+judgmentSearch.formField.chooseScDivision = Dowolny wydział izby
+judgmentSearch.formField.chooseScJudgmentForm = Dowolna forma
+judgmentSearch.formField.chooseScPersonnelType = Dowolny typ
 judgmentSearch.tooltip.settings = Pokaż ustawienia wyszukiwania
 
+judgmentSearch.settings.iconAlt = Ikona ustawień wyszukiwania
```

Comment:
Are you sure? Think about it, what does it give a user? In my view what you're talking about is an 'informative image', but here we have an image that performs specific action, a kind of button, so I guess that the user is more interested in that action rather than what the icon is.

See (especially the 5. point):
https://www.cs.tut.fi/~jkorpela/html/alt.html

But I am not an expert so I don't want to insist.